src/components/company/ZoomChat.js

```diff
@@ -14,9 +14,17 @@ export default ({ webcamPhotos }) => (
     </div>
 
     <div className={styles.grid}>
-      {webcamPhotos.map((employee, idx) => (
-        <div key={idx} className={styles.photo} style={{ backgroundImage: `url(${employee.node.webcamPhoto.file.url})`}}></div>
-      ))}
+      {webcamPhotos.map((employee) => {
+        const { slug, webcamPhoto } = employee.node;
+        return (
+          <img
+            key={slug}
+            className={styles.photo}
+            src={webcamPhoto.resize.src}
+            alt=""
```

Comment:
would it make sense to put the employee name for the alt text?

Reply:
Yep, thanks!

Reply:
7ca9d4c